pkg/application/create.go

```diff
@@ -59,17 +62,24 @@ func checkoutNewBranch(localRepo *git.LocalRepository, branchName string) error
 	if currentBranch != git.MainBranch {
 		if err := localRepo.CheckoutBranch(&git.CheckoutOp{
 			BranchName: git.MainBranch,
+			DryRun:     false,
 		}); err != nil {
 			return err
 		}
 	}
 	return localRepo.CheckoutBranch(&git.CheckoutOp{
 		BranchName:      branchName,
 		CreateIfMissing: true,
+		DryRun:          false,
 	})
 }
 
 func (svc *Service) Create(op CreateOp) (result CreateResult, err error) {
+	log.Info().
```

Comment:
I would vote for passing logger instance explicitly instead of having a global variable.
While logger seems like a minor thing, this might be a problem later

Reply:
Having to pass the logger around every time you want to log something will discourage people using it, and we'll have +1 arg for every function that wants to write logs, it would be worth finding out what the team wants